lib/Froxlor/Cron/Dns/DnsBase.php

```diff
@@ -202,15 +204,15 @@ public function writeDKIMconfigs()
 					$max_dkim_id_stmt = Database::query("SELECT MAX(`dkim_id`) as `max_dkim_id` FROM `" . TABLE_PANEL_DOMAINS . "`");
 					$max_dkim_id = $max_dkim_id_stmt->fetch(\PDO::FETCH_ASSOC);
 					$domain['dkim_id'] = (int) $max_dkim_id['max_dkim_id'] + 1;
-					$privkey_filename = \Froxlor\FileDir::makeCorrectFile(Settings::Get('dkim.dkim_prefix') . '/dkim_' . $domain['dkim_id']);
```

Comment:
The file is not up to date with the current master branch, the underscore has been removed (and not replaced by a dash)